### PR TITLE
Auto assign3.0

### PR DESCRIPTION
**Pull request by RamiyaRatnarajah**

### CODE REVIEW

File: src/main/java/com/kajan/iworkflows/controller/FileController.java [hunk 1]
```diff
@@ -28,28 +31,48 @@
 
     private NextcloudService nextcloudService;
     private DummyUserStoreServiceImpl userStoreService;
+    private LearnOrgServiceImpl learnOrgService;
+
+    @Value("${testing}")
+    private Boolean testing;
```
Comment (KajanM) [inline src/main/java/com/kajan/iworkflows/controller/FileController.java]: @RamiyaRatnarajah  did you check the file upload in test mode and with learnorg service?

File: src/main/java/com/kajan/iworkflows/service/impl/LearnOrgServiceImpl.java [hunk 2]
```diff
@@ -25,33 +26,51 @@
...
 
-        HttpHeaders headers = new HttpHeaders();
-        headers.setContentType(MediaType.APPLICATION_FORM_URLENCODED);
         MultiValueMap<String, String> map = new LinkedMultiValueMap<String, String>();
         map.add("access_token", accesstoken);
-        HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<MultiValueMap<String, String>>(map, headers);
-        log.debug("uri : " + userInforUri);
-        ResponseEntity<UserStore> response = this.restTemplate.postForEntity(userInforUri, request, UserStore.class);
+        HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<MultiValueMap<String, String>>(map, getLearnOrgHeaders());
+        log.debug("uri : " + userInfoUri);
```
Comment (KajanM) [inline src/main/java/com/kajan/iworkflows/service/impl/LearnOrgServiceImpl.java]: don't use +, instead, use interpolation like `log.debug("uri : {} ", userInforUri)`, {} will be automatically replaced by userInfoUri

File: src/main/java/com/kajan/iworkflows/controller/FileController.java (hunk 1 above)
Comment (RamiyaRatnarajah) [reply src/main/java/com/kajan/iworkflows/controller/FileController.java]: ya checked. Its uploaded in nextcloud. But error is coming in handleAttachment class.

Comment (KajanM) [reply src/main/java/com/kajan/iworkflows/controller/FileController.java]: Can you post the error?

File: src/main/java/com/kajan/iworkflows/service/impl/LearnOrgServiceImpl.java (hunk 2 above)
Comment (RamiyaRatnarajah) [reply src/main/java/com/kajan/iworkflows/service/impl/LearnOrgServiceImpl.java]: ok

**Pull request merged**